.editorconfig

```diff
@@ -10,5 +11,5 @@ trim_trailing_whitespace = true
 [tests/Fixtures/**]
 trim_trailing_whitespace = false
 
-[{*.yml, *.yaml}]
+[*.{yml,yaml,md,json}]
```

Comment:
Not all existing files following this rule. Mind to apply it first ?

Reply:
# 🏃🏻‍♂️

Reply:
Done 🍏